src/config/2021.json

```diff
@@ -579,6 +579,62 @@
       ],
       "avatar_url": "https://avatars.githubusercontent.com/u/9135107?v=4&s=200",
       "github": "ydimova"
+    },
+    "alonko": {
+      "name": "Alon Kochba",
+      "teams": [
+        "leads",
+        "authors"
+      ],
+      "avatar_url": "https://avatars.githubusercontent.com/u/7407047?v=4&s=200",
+      "github": "alonkochba",
+      "linkedin": "alonkochba",
+      "twitter": "alonkochba"
+    },
+    "hyperpress": {
+      "name": "John Teague",
+      "teams": [
+        "reviewers"
+      ],
+      "avatar_url": "https://avatars3.githubusercontent.com/u/25212653?v=4&s=200",
+      "website": "https://gemservers.com",
+      "github": "logicalphase",
+      "twitter": "jtteag"
+    },
+    "andreylipattsev": {
+      "name": "Andrey Lipattsev",
+      "teams": [
+        "reviewers"
+      ],
+      "avatar_url": "https://avatars.githubusercontent.com/u/16627210?v=4&s=200",
+      "github": "andreylipattsev",
+      "twitter": "AndreyLipattsev"
+    },
+    "alankent": {
+      "name": "Alan Kent",
+      "teams": [
+        "reviewers"
+      ],
+      "avatar_url": "https://avatars0.githubusercontent.com/u/5702163?v=4&s=200",
+      "website": "https://alankent.me",
+      "github": "alankent",
+      "twitter": "akent99"
+    },
+    "chrissater": {
+      "name": "Chris Sater",
+      "teams": [
+        "reviewers"
+      ],
+      "avatar_url": "TODO",
+      "linkedin": "christophersater"
+    },
+    "tosinarasi": {
+      "name": "Tosin Arasi",
+      "teams": [
+        "analysts"
+      ],
+      "avatar_url": "TODO",
```

Comment:
@rviscomi I filled in most of the contributors, couldn't find a github image for @TosinArasi / @saterchris  though
if you guys have a link to an image on github that would be great.. maybe it can be non-github too but all the examples here are github avatars

Reply:
Just right click on the image in their profile and view image. And then add `&200`

@TosinArasi - https://avatars.githubusercontent.com/u/89331243?v=4&s=200
@saterchris  - https://avatars.githubusercontent.com/u/62304935?v=4&s=200

The fact that they are the default GitHub avatars is fine. At least they have the ability to change them if they want.

Reply:
Done